### PR TITLE
CMR-7443

### PR DESCRIPTION
**Pull request by ygliuvt**
This PR is for work done mostly by @abbottry and @macrouch to add integration test support and tidy up the jtest tests and codebase.

### CODE REVIEW

File: graph-db/serverless.yml [hunk 1]
```diff
@@ -82,3 +76,10 @@ resources:
         PolicyName: cmrParamStorePolicy-${opt:stage}
         Roles:
           - Ref: IamRoleGraphLambdaExecution
+custom:
+  variables:
+    # Default values for environment variables
+    CMR_ROOT: ${env:CMR_ROOT, 'https://cmr.sit.earthdata.nasa.gov'}
```
Comment (jaybarra) [inline graph-db/serverless.yml:82]: I would suggest this is just blank if no default is set to make sure its being done explicitly as part of a deployment

File: graph-db/serverless/src/utils/cmr/__test__/fetchPageFromCmr.test.js [hunk 2]
```diff
@@ -1,44 +1,36 @@
...
-})
-
-afterEach(() => {
-  // Restore any ENV variables overwritten in tests
-  process.env = OLD_ENV
-})
+import * as indexPageOfCmrResults from '../indexPageOfCmrResults'
 
 describe('fetchPageFromCMR', () => {
+  // beforeAll(() => {
```
Comment (jaybarra) [inline graph-db/serverless/src/utils/cmr/__test__/fetchPageFromCmr.test.js:8]: Can the comments be removed?

**Review by jaybarra — APPROVED** [review]

File: graph-db/serverless.yml (hunk 1 above)
Comment (ygliuvt) [reply graph-db/serverless.yml:82]: I am going to change it to be deduced from `stage`.

**Review by macrouch — APPROVED** [review]

File: graph-db/serverless.yml (hunk 1 above)
Comment (ygliuvt) [reply graph-db/serverless.yml]: Actually, we do want the ability to set a `CMR_ROOT` that is different from what `stage` maps to. And I don't want to have to put in `CMR_ROOT` every time for the default deployment `sit`. So I am going to keep it the way it was.

**Pull request merged**